src/components/Card/Card.js

```diff
@@ -1,28 +1,92 @@
 import { useState } from 'react';
 
+import { MdEdit, MdSave, MdEditOff } from 'react-icons/md';
+
 import './Card.css';
 
 function Card(props) {
+  //checkbox
   const [isChecked, setIsChecked] = useState(false);
 
+  //edit
+  const [isEditing, setIsEditing] = useState(false);
+
+  const [title, setTitle] = useState(props.title);
+  const [editedTitle, setEditedTitle] = useState('');
+
+  const [descr, setDescr] = useState(props.descr);
+  const [editedDescr, setEditedDescr] = useState('');
+
+  //checkbox func
   const checkboxChangeHandler = () => {
     setIsChecked(!isChecked);
   };
 
+  //buttons func-s
+  const clickEditButtonHandler = () => {
+    setIsEditing(!isEditing);
+    setIsChecked(isChecked ? !isChecked : isChecked);
+    setEditedTitle(title);
+    setEditedDescr(descr);
+  };
+
+  const clickSaveButtonHandler = () => {
+    setIsEditing(!isEditing);
+    setTitle(editedTitle);
+    setDescr(editedDescr);
+  };
+
+  const clickCancelButtonHandler = () => {
+    setIsEditing(!isEditing);
+    setEditedTitle(title);
+    setEditedDescr(descr);
+  };
+
+  //clasnames variables
   const cardClassName = isChecked ? 'card checked' : 'card';
+  const buttonClassName = isEditing ? 'card-block__edit-btn hide' : 'card-block__edit-btn';
+  const checkboxClassName = isEditing ? 'card-block__checkbox hide' : 'card-block__checkbox';
```

Comment:
Please install the ClassNames library and try to use it to switch the only 'hide' class on and off.